packages/mui-base/src/useList/useList.ts

```diff
@@ -28,13 +28,15 @@ import { EventHandlers } from '../utils/types';
 const EMPTY_OBJECT = {};
 const NOOP = () => {};
 
-const defaultItemComparer = <ItemValue>(optionA: ItemValue, optionB: ItemValue) =>
-  optionA === optionB;
+function defaultItemComparer<ItemValue>(optionA: ItemValue, optionB: ItemValue) {
```

Comment:
Is this related to vitest changes?

Reply:
Yes, vitest can't execute jsx in js files. There's a workaround, but you have to have every file interpreted as jsx. That unfortunately breaks this code.